tests/resources/bigquery_test_tables.sql

```diff
@@ -141,6 +140,31 @@ INSERT INTO `pso_data_validator`.`dvt_char_id` VALUES
 ('DVT4  ', 'Row 4  	  '),
 ('DVT5  ', 'Row 5');
 
+# BigQuery does not have a fixed char data type and blanks at the end of strings are not removed by the database.
+# Other databases have a fixed char data type and these tables are to allow the comparison of those columns with BQ and each other
+CREATE OR REPLACE TABLE `pso_data_validator`.`dvt_varchar_id`
```

Comment:
We already have test table `dvt_string_id`. Do we either not need `dvt_varchar_id ` or need to remove `dvt_string_id`?

Reply:
You've removed the tests that use `dvt_string_id` so I assumed the new table was a replacement for the old one. If they serve different purposes should the old tests be left in place? If the table is redundant we can remove it from the source code but not actually drop it until your changes reach `develop`.